### DISCUSSION TITLE
long range forecast?

### DISCUSSION THREAD

**Discussion by dangelsaurus** [weather-gov/api#674] [Q&A]
Is there a reason there's no extended (10 or 14 day) forecast available? Or is there a good upstream source to get this data? I came here when DarySky was officially changed to Apple Weather, as the Terms of Services weren't agreeable for what I'm doing. But I sure do miss the extended forecast. 

(sorry I know this is sorta off topic, but you folks have been extremely helpful at pointing me in the right direction in the past with questions like this.

**Comment by sullynole** [comment]
api.weather.gov provides the gridded forecasts from local weather forecast offices across the United States and interests: https://www.weather.gov/srh/nwsoffices

These forecasts created by local meteorologists extend out 7 days so there is no data that exists beyond what is available here.  If you're looking for extended forecasts beyond 7 days that will require model data.  NBM would be a good option through 10 days but if looking for a lot more than that, probably going to require global model data.  https://nomads.ncep.noaa.gov
